examples/rich-text/index.js

```diff
@@ -70,7 +70,9 @@ class RichText extends React.Component {
 
   hasMark = (type) => {
     const { state } = this.state
-    return state.marks.some(mark => mark.type == type)
+    const characterMarks = state.marksPerCharacter
+    return characterMarks.length &&
+      characterMarks.every(charMarks => charMarks.some(mark => mark.type == type))
```

Comment:
If we do do that, I'm not sure if it's worth it to have the `marksPerCharacter` anymore, since people can just do `state.characters.every(c => c.marks.some` for their own custom cases, which I think would be just as expensive?